lib/models/device.coffee

```diff
@@ -1070,8 +1085,13 @@ getDeviceModel = (deps, opts) ->
 	# 	if (error) throw error;
 	# });
 	###
-	exports.update = (uuidOrId, options, callback) ->
-		exports.get(uuidOrId).then (device) ->
+	exports.update = (uuidOrId, options = {}, callback) ->
```

Comment:
What happens when updating with an empty object? Does it serve like a ping?

Reply:
Update options are actually only allowed to be `force: true/false` (or nothing). All this method does is ping a device to get it to check for an update. I'm not 100% sure why that's useful, but it's a feature the supervisor supports, so I'm happy to leave it here for now.